.github/CODEOWNERS

```diff
@@ -1,2 +1,6 @@
 # default reviewers
 *                 @greenbone/openvas-scanner-maintainers
+
+# devops
+.github/*         @greenbone/devops
+.docker/*         @greenbone/devops
```

Comment:
In https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners there is the example:

```
# The `docs/*` pattern will match files like
# `docs/getting-started.md` but not further nested files like
# `docs/build-app/troubleshooting.md`.
docs/*  docs@example.com
```
You probably want `.github/`. Although there is no example in the doc about hidden folders.

Reply:
n4rf.